locale/es-ES.json

```diff
@@ -0,0 +1,167 @@
+    "tax_disclaimer": "* Incluye VAT más costo de envíos",
+    "item_added": "{count, plural, =0 {# no se agregaron artículos} =1 {{count} artículo agregado} other {{count} artículos agregados}}",
+    "add_to_cart": "Agregar al carrito",
+    "go_to_cart": "Ir al carrito"
+  },
+  "category": {
+    "error": {
+      "title": "Oops",
+      "not_found": "No se encontró la categoría"
+    },
+    "no_result": {
+      "heading": "Oops!",
+      "body": "No se encontraron productos."
+    }
+  },
+  "search": {
+    "label": "Buscar",
+    "cancel": "Cancelar",
+    "no_result": {
+      "heading": "Oops!",
+      "body": "Su búsqueda \"{searchPhrase}\" tiene 0 resultado."
+    }
+  },
+  "shipping": {
+    "unknown": "más envío",
+    "free": "Entrega gratuita",
+    "free_short": "Gratis",
+    "cost": "{price} envío"
+  },
+  "liveshopping": {
+    "discount_badge": "AHORRE {0}%"
+  },
+  "modal": {
+    "confirm": "Confirmar",
+    "dismiss": "Descartar",
+    "ok": "Ok",
+    "title_error": "Oops!",
+    "body_error": "Ocurrió un error inesperado. vuelva a intentarlo."
+  },
+  "cart": {
+    "subtotal": "Subtotal",
+    "checkout": "Pagar",
+    "remove": "Quitar",
+    "edit": "Editar",
+    "empty": "Su carrito está vacío",
+    "continue": "Continuar comprando",
+    "redeem_coupon": "Ingresar código del cupón",
+    "coupon_code": "Código",
+    "default_coupon_label": "Cupón",
+    "coupon_was_added": "¡El cupón fue agregado al carrito!",
+    "coupons_not_supported": "Agregue los cupones y vales de regalo mientras realiza el pago",
+    "free_shipping": "Envío gratis"
+  },
+  "login": {
+    "headline": "Bienvenido",
+    "subline": "Inicie sesión para continuar",
+    "email": "Email",
+    "password": "Contraseña",
+    "forgot_password": "¿Olvidó su contraseña?",
+    "forgot_password_popup": "Utilice el sitio web para restablecer su contraseña.",
+    "button": "Iniciar sesión",
+    "no_account": "¿Aún no tiene una cuenta?",
+    "signup": "Registro",
+    "logout_message": "Ha sido desconectado.",
+    "error": "Su usuario y contraseña no corresponden a ninguna cuenta"
+  },
+  "navigation": {
+    "login_register": "Iniciar sesión / Registro",
+    "your_account": "Su cuenta",
+    "home": "Inicio",
+    "categories": "Categorías",
+    "my_orders": "Mis pedidos",
+    "cart": "Carrito de compras",
+    "shipping": "Métodos de envío",
+    "about": "Sobre",
+    "contact": "Contacto",
+    "payment": "Métodos de pago",
+    "terms": "Términos y Condiciones",
+    "privacy": "Política de Privacidad",
+    "welcome_message": "Bienvenido {name}",
+    "return_policy": "Política de devolución",
+    "logout": "Cerrar sesión",
+    "store_information": "Store Information",
+    "more_menu": "Más"
+  },
+  "reviews": {
+    "rating": "Calificación",
+    "no_reviews": "Sea el primero en calificar este producto:",
+    "review_count": "({count} {count, plural, =1 {Comentario} other {Comentarios}})",
```

Comment:
at least this is still wrong. Please make really really sure everything is correct and double check everything, because we take a look.